src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split('\n') // array key-value
+    .map(row => row.split(': ')) // array [key,value]
```

Comment:
What if we will receive styles without space between property and value?)

Reply:
Cry... joke ;)
It`s misprint.